api.md

```diff
@@ -327,10 +357,8 @@ SELECT create_hypertable('conditions', 'time', if_not_exists => TRUE);
 
 #### Best Practices [](create_hypertable-best-practices)
 
-Users of TimescaleDB often have two common questions:
-
-1. How large should I configure my intervals for time partitioning?
-1. Should I use space partitioning, and how many space partitions should I use?
+One of the most common questions users of TimescaleDB have revolve around
```

Comment:
"revolve" >> "revolves"?  As in "one question users have revolves around..."

Reply:
Done!